### ISSUE TITLE
Installing via slack app directory doesn't work

### ISSUE THREAD

**Issue by jakubsuchybio** [jlmart88/kaomoji-slack#34]
Clicking on "Add to slack" here: https://btl-pu4-sw.slack.com/apps/A3YF6DAG0-kaomoji?tab=more_info doesn't work: 
![image](https://github.com/user-attachments/assets/66ab2159-0b8e-4222-a0a9-15b7a01a12ff)

**Comment by jlmart88** [comment]
Thanks for the report. I've submitted an update to the Slack app directory; in the meantime you can install from the [homepage](https://kaomoji.io)

**Comment by jakubsuchybio** [comment]
Yep, already found the second install option, and already installed. Btw why isn't it usable in threads? :(
